app/jobs/send_client_save_ctc_letter_message_job.rb

```diff
@@ -0,0 +1,15 @@
+class SendClientSaveCtcLetterMessageJob < ApplicationJob
+  def perform(client)
+    return if client.intake.email_notification_opt_in != "yes" && client.intake.sms_notification_opt_in != "yes"
+
+    service_name = client.intake.is_ctc? ? "GetCTC" : "GetYourRefund"
```

Comment:
This may be overkill since we only use the service_name in here, but it might make sense to use MultiTenetService to achieve rendering the appropriate name based on the service if we're doing things like this in the future (would have been especially helpful if this email had a link or something)

Reply:
Oh good point! Making the change to include `MultiTenantService.new(:ctc).service_name` for the name instead of hard coding it. The ticket didn't specify to make it a link but I can run it by Yvonne